site-src/concepts/versioning.md

```diff
@@ -92,6 +92,45 @@ change across API versions.
 ### Major version (e.g. v0.x to v1.0)
 * There are no API compatibility guarantees when the major version changes.
 
+## Graduation Criteria
+
+### Resources
+
+#### Alpha -> Beta
+A resource to graduate from alpha to beta must meet the following criteria:
+
+* Implemented by several implementations
+* Initial conformance tests are in place
+* Validation is well thought out
+* Most of the API surface is being exercised by users
+* Approval from subproject owners + KEP reviewers
+
+#### Beta -> GA
+
+A resource to graduate from beta to GA must meet the following criteria:
+
+* Thorough conformance test coverage 
```

Comment:
Similar feedback for `thorough` as https://github.com/kubernetes-sigs/gateway-api/pull/1136/files#r863232847.

Reply:
Updated, thanks!